components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -685,6 +686,11 @@ protected void concludeFlow(HttpServletRequest request, HttpServletResponse resp
             publishAuthenticationSuccess(request, context, sequenceConfig.getAuthenticatedUser());
         }
 
+        // Passing the federated tokens to the authentication result.
+        if (context.getProperty(FEDERATED_TOKENS) != null) {
```

Comment:
Lets add unit test to make sure that available federated tokens are getting added to the authresult